src/Install/DatabaseConfig.php

```diff
@@ -47,7 +47,7 @@ public function toArray()
             'collation' => 'utf8mb4_unicode_ci',
             'prefix'    => $this->prefix,
             'strict'    => false,
-            'engine'    => 'InnoDB',
+            'engine'    => NULL,
```

Comment:
Again, why mark this as null and not tell it to use innodb?

Reply:
The issue I had was the Database did not support **MyISAM**. Wouldn't changing it to **Innodb** just bake in the same issue for a future version of me, with a Database that does not support **Innodb**. As such I determined that NULL (or no default) was the best option. Leaving it up to the Laravel Backend to determine.